code/modules/mob/living/life.dm

```diff
@@ -110,18 +110,13 @@
 
 /mob/living/proc/handle_traits()
```

Comment:
Who keeps making this proc? It's like having a handle_flags

Reply:
Beats me, can’t really rename it to handle_eyes because it’s also used to send signals for brain damage on carbons